src/components/Header/UserMenu.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react'
+import { isLoggedIn, getUser } from '../../services/util'
 
 const UserMenu = () => {
+  let greetingMessage = `مرحباً ${getUser().username.toUpperCase()} `
```

Comment:
make this defensive (maybe use destructuring) .. otherwise it could break the app ifmake sure if `getUser()` or `.username` are null

Reply:
@kabaros 
I make the check in src\components\Header\Navbar.js
`<li className="mr-6 my-2 md:my-0">
          {isLoggedIn() ? (
            <>
              <UserMenu />
            </>
          )`